src/lib/time.ts

```diff
@@ -0,0 +1,33 @@
+function getAllElapsedMinutes(dateString: string) {
+  const date = new Date(dateString);
+  const diff = Date.now() - date.getTime();
+
+  return Math.floor(diff / 1000 / 60);
+};
+
+function getElapsedSeconds(dateString: string) {
+  const date = new Date(dateString);
+  const diff = Date.now() - date.getTime();
+
+  return Math.floor(diff / 1000 % 60);
+};
+
+export const elapsed = {
+  allMinutes: getAllElapsedMinutes,
+  seconds: getElapsedSeconds,
+};
+
+function formatXX(item: number) {
+  return item < 10 ? '0' + item : item;
+}
+
+export function format(timestamp: string) {
```

Comment:
I'd recommend to define an [opaque](https://github.com/sindresorhus/type-fest/blob/main/source/opaque.d.ts) `Timestamp` type to make types stronger:

```ts
type Timestamp = Opaque<string, 'ISO timestamp'>
```

Reply:
I do not think that this will add reliability, but I probably will use it later